osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -28,7 +53,21 @@ protected override double StrainValueOf(DifficultyHitObject current)
             var osuCurrent = (OsuDifficultyHitObject)current;
 
             double result = 0;
-
+            if (osuCurrent.JumpDistance < 150)
+            {
+            double sectionvelocity = osuCurrent.JumpDistance/osuCurrent.StrainTime;
+            string tempstor = Path.GetTempPath();
+            if (sectionvelocity < streamaimconst) 
+                { 
+                    string comma = "" ;
+                    string JumpDistanceArrayn = osuCurrent.JumpDistance.ToString() + comma;
+                    string StrainTimeArrayn = osuCurrent.StrainTime.ToString() + comma;
+                    string jdapath = tempstor + "jda.txt";
+                    string jda2path = tempstor + "jda2.txt";
+                    System.IO.File.WriteAllText(jdapath, JumpDistanceArrayn);
+                    System.IO.File.WriteAllText(jda2path, StrainTimeArrayn);
```

Comment:
this is overwriting the file for every note, the file only contains the last value

Reply:
Now this one is quite the problem I've been facing, I tried doing recursive addition to arrays but it doesn't work ( It doesn't return arrays of data). Somehow when this function is called it doesn't add new values to my Array. Could you suggest a way to prevent an overwrite or seperate the new values with comma?

Reply:
Use [`List<T>`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1?view=netcore-3.1)

Reply:
Fixed in #8 .